### PR TITLE
docs: update `client.webSocketURL` examples

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [X] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
NA

### Motivation / Use-Case
update docs.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: examples/cli/web-socket-url-cli/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # CLI: Web Socket URL
 
 ```console
-npx webpack serve --open-target --host 0.0.0.0 --web-socket-url <insert-host>:8080
+npx webpack serve --open-target --host 0.0.0.0 --client-web-socket-url <insert-host>:8080
```
Comment (alexander-akait) [inline examples/cli/web-socket-url-cli/README.md:4]: it should have `ws://` prefix, i.e. full valid URL

Comment (snitin315) [reply examples/cli/web-socket-url-cli/README.md]: fixed

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
@snitin315 Let's add `test:examples` script where we test all our examples

**Comment by snitin315** [comment]
> @snitin315 Let's add `test:examples` script where we test all our examples

Good Idea, Will add 👍🏻

**Pull request merged**